modules/WordCloudPlugin/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <groupId>org.gephi</groupId>
+        <version>0.10.0</version>
+    </parent>
+
+    <groupId>exploreyourdata</groupId>
+    <artifactId>word-cloud-plugin</artifactId>
+    <version>0.10.0</version>
```

Comment:
I wouldn't recommend to use Gephi's versions for a plugin. Why not `1.0.0.`?

Reply:
Indeed